Makefile

```
@@ -281,8 +246,7 @@ ${GO_MOD_OUTDATED}:
 	$(call go-build,./vendor/github.com/psampaz/go-mod-outdated)
 
 ${GOLANGCI_LINT}:
-	export \
-		VERSION=v1.30.0 \
+	VERSION=v1.32.2 \
```

We still need the `export` here to pass the `BINDIR` to the underlying script:
```suggestion
	export VERSION=v1.32.2 \
```